lib/recurly/client.php

```diff
@@ -141,6 +149,14 @@ private function _sendRequest($method, $uri, $data = '')
       curl_setopt($ch, CURLOPT_CUSTOMREQUEST, $method);
     }
 
+    if(Recurly_Client::$logger){
+      Recurly_Client::$logger->info('Send request to Recurly', array('method' => $method, 'uri' => $uri));
+      Recurly_Client::$logger->debug('Send request with headers', array('headers' => $headers));
+      if(!empty($data)){
```

Comment:
@drewish , you're absolutely correct. This is a defensive programming habit that I have from working with globals. In this case $data has a default argument value, so it should be changed. `if ($data) { ... }` is faster and looks nicer.